jujugui/static/gui/src/app/components/header-breadcrumb/header-breadcrumb.js

```diff
@@ -138,9 +138,9 @@ YUI.add('header-breadcrumb', function() {
       @method _generateUserLink
     */
     _generateUserLink: function() {
-      const auth = this.props.authDetails;
-      if (auth && (auth.user || auth.loading)) {
-        return this._buildProfile(auth.loading ? '...' : auth.rootUserName);
```

Comment:
The `auth` object there had no `loading` attribute, so the ellipsis was never shown. So I just removed it. At this point once you've hit the canvas you're logged into a controller, even if the model is still loading.